features/manage-position/Create.tsx

```diff
@@ -159,6 +164,17 @@ const Create = () => {
       }
     };
 
+    const setBackingCollateralToMin = () => {
+      // By default set amount of collateral to the minimum required by the GCR constraint. This
+      // default is intended to encourage users to maximize their capital efficiency.
+      const minBackingCollateral = gcr * Number(tokens);
+      setCollateral(minBackingCollateral.toString());
+    };
+
+    useEffect(() => {
+      setBackingCollateralToMin();
+    }, [gcr, tokens, minSponsorTokensFromWei]);
```

Comment:
If this gets updated anytime the gcr changes, does that mean that if you're looking at this page as someone else is creating or redeeming a position, the collateral field might update unexpectedly?

There may not be a better way to do it, but just wanted to make sure I'm understanding this correctly.

Reply:
I believe that's right, we could basically remove `gcr` from this so that it just updates once on render but I see this as an edge case that shouldn't cause problems

Reply:
Yeah, I think this is fine, and I don't see an easy way to avoid it unless you _only_ fill it when the tokens field is modified.